types/heic-convert/package.json

```diff
@@ -13,5 +13,9 @@
             "name": "Tregan",
             "githubUsername": "treemmett"
         }
-    ]
+    ],
+    "exports": {
+        ".": "./index.d.ts",
+        "./browser": "./browser.d.ts"
+    }
```

Comment:
heic-convert's package.json doesn't have an `exports` entry so the types shouldn't either.

Reply:
Yup, but they have a section with the missing export: https://github.com/catdad-experiments/heic-convert?tab=readme-ov-file#usage-in-the-browser

I guess since their package is commonjs there is no problem using require('heic-convert/browser') but when using import convert from 'heic-convert/browser' there is an error about missing type definitions, that's why I'm adding the exports field